ably-ios/ARTRealtime+Private.h

```diff
@@ -25,21 +25,17 @@ ART_ASSUME_NONNULL_BEGIN
 @property (readonly, strong, nonatomic) NSMutableArray *stateSubscriptions;
 
 // Transport Events
-- (void)onHeartbeat:(ARTProtocolMessage *)message;
-- (void)onConnected:(ARTProtocolMessage *)message withErrorInfo:(art_nullable ARTErrorInfo *)errorInfo;
-- (void)onDisconnected:(ARTProtocolMessage *)message;
-- (void)onError:(ARTProtocolMessage *)message withErrorInfo:(art_nullable ARTErrorInfo *)errorInfo;
+- (void)onHeartbeat;
+- (void)onConnected:(ARTProtocolMessage *)message;
+- (void)onDisconnected;
```

Comment:
`onClosed` doesn't exist right from the beginning. I'm assuming the tests are calling the `close()` method.